projects/pluto/README.md

```diff
@@ -0,0 +1,37 @@
+# Pluto
+
+The service is a Spring Boot service playing role of a lightweight API gateway using Spring Cloud Gateway.
+
+For a detailed description of the service, see the [Fairspace documentation](../../README.adoc).
+
+## Development
+
+### Running the app in development mode
+
+Prerequisites:
+
+- Java 21
+- Gradle
+- Running Saturn service
+- Running Keycloak instance
+- Running PostgresQL instance (optional)
+- Environment variable `KEYCLOAK_CLIENT_SECRET` set to `**********`
```

Comment:
The KEYCLOAK_CLIENT_SECRET value depends on the local settings (`**********` is a default)

Reply:
updated